doc/changing_derivatives.md

```diff
@@ -59,12 +67,17 @@ class ImageUploader < Shrine
 end
 ```
 
-Note that we cannot update our attachment URLs yet, because only new
-attachments will have thumbnails generated, existing attachments will still
-have only the original file. So we can go ahead and deploy this change.
+*Note: we cannot update our attachment URLs yet, because only new
+attachments will have thumbnails generated, existing attachments will 
+only have the original file.*
+
+We can deploy this change to production.
 
-Next, we want to generate derivatives for all existing attachments in
-production. We can do this by running the following script:
+#### 2. Generate derivatives for existing attachments
+
+To generate derivatives for all existing attachments in production, run
+the following script. It fetches the photos in batches, downloads the image 
+in `:store` storage, creates derivatives, and persist the changes. 
```

Comment:
I would prefer using the more high-level "permanent storage"  term here, so that it explains the conditional.